test/components/tree-view.spec.tsx

```diff
@@ -2,13 +2,11 @@ import * as React from 'react';
 import { expect, ClientRenderer, sinon, simulate, waitFor } from 'test-drive-react';
 import { TreeView, TreeItem } from '../../src';
 import { TreeViewDemo, treeData } from '../../demo/components/tree-view-demo';
```

Comment:
I have a bad feeling about mixing the treeData from the demo with unit tests. I think it should be separated for finer control of tests